Search/ObjectToDocumentConverter.php

```diff
@@ -110,8 +115,16 @@ public function objectToDocument(IndexMetadata $metadata, $object)
             $document->setLocale($locale);
         }
 
+        $this->blockValues = [];
```

Comment:
As written above this should just be a local variable and not a class property. To keep service stateless.